toolkit/tools/imagecustomizerapi/veritycorruptionresponse.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type VerityCorruptionResponse string
+
+const (
+	VerityCorruptionResponseIgnore  VerityCorruptionResponse = "ignore-corruption"
```

Comment:
Change to `"ignore"`, `"restart"`, and `"panic"` to be consistent with the imager's config.